requirements.txt

```diff
@@ -11,6 +11,7 @@ backports.ssl-match-hostname==3.5.0.1  # via apache-libcloud
 caslib.py==2.2.2          # via django-cyverse-auth
 cffi==1.10.0              # via cryptography
 contextlib2==0.5.5        # via raven
+django-cors-headers==1.3.1
```

Comment:
Connor did mention that (taught me what they were). Luckily we don't actually need them. I'll remove them from the PR.

Whitenoise is a way serve static files in production, without an nginx proxy. I thought I might need it for a local development setup, but turns out I don't. I'll remove it.

The `django-cors-headers` module was dropped in to resolve the issue of serving requests across two domains (localhost:8000 and localhost:8080) but it turns out to be the webpack-dev-server that need to add CORS headers, not Troposphere. I'll remove that as well.